src/server.ts

```diff
@@ -26,16 +26,7 @@ app.use(
 );
 
 // Handle logs in console during development
-app.use(
-  morgan('dev', {
-    skip: (req, res) => {
-      // logs everything but health checks on dev, errors only otherwise
-      return process.env.NODE_ENV === 'development' || process.env.DEBUG === 'true'
-        ? [ARRANGER_READY_ENDPOINT, HEALTH_ENDPOINT].includes(req.originalUrl)
-        : res.statusCode < 400;
-    },
-  }),
-);
```

Comment:
Its dev... just log everything :)

Reply:
in this case, it is certainly not what you want to do... as logging everything would include an endless barrage of health pings in the logs.